packages/client/test/sim/README.md

```diff
@@ -0,0 +1,45 @@
+### Shandong sim setup
+
+This sim test is to setup a single ethereumjs<>lodestar instance to allow executing testvectors for the EIPs targetting Shanghai hardfork.
+
+### Prerequisite(s)
+
+- Docker since lodestar docker image is used to run CL to drive the post-merge sim run
+- Docker should run without `sudo`, else prefix `sudo` in from of docker commands in `test/sim/single-run.sh`
+
+### How to run
+
+1. `npm install` the `ethereumjs-monorepo`
+2. `cd packages/client`
+3. `docker pull chainsafe/lodestar:latest`
+4. Install package `jq` is not installed
```

Comment:
Short nit: might be nice to add "Linux package" here, I was first unsure if there is an npm package with that name that I should install globally.